README.md

```diff
@@ -32,12 +38,25 @@ aws cloudformation deploy \
 TODO: Add example of exporting an existing network
 
 ### Infrastructure stack
+
+__Parameters:__
++ NetworkStackName
+  + Description: The name of the parent networking stack created.
+  + Default: "mellon-network"
++ EnvironmentName
+  + Description: Any value that describes where the service exists
+  + Default: "dev"
++ DomainName
+  + Description: 'The DomainName to be used for all entities to be created. All services will be built within this Domain'
+  + Default: 'library.nd.edu'
+
 Note: This will require adding a DNS entry to validate the certificate created by the stack. The stack will not complete until this is done. See https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-validate-dns.html.
 
 ```console
 aws cloudformation deploy \
   --capabilities CAPABILITY_IAM \
   --template-file deploy/cloudformation/app-infrastructure.yml \
+  --parameter-overrides NetworkStackName='unpeered-network' DomainName='libraries.nd.edu' \
```

Comment:
Again, the instructions here are meant to be highly general - not specific to our configuration. I think this requires some updating to reflect that.

Reply:
Right. I got a bit of a mixed message on this. I wanted Jon to know of the changes made for DNS so that his next rebuild of the full infrastructure would include the DNS information. He suggested the README as a place holder, at least for now.

Also, there is an entirely different Jira ticket for overhauling the README (MEL-175). My primary goal was to point out the changes.

Reply:
OK. I think I understand. Definitely would like some others to weigh in though.